src/Explorer/Panes/AddCollectionPanel.tsx

```diff
@@ -1312,7 +1312,7 @@ export class AddCollectionPanel extends React.Component<AddCollectionPanelProps,
   }
 
   private shouldShowVectorSearchParameters() {
-    return isVectorSearchEnabled() && this.shouldShowCollectionThroughputInput();
+    return isVectorSearchEnabled() && (isServerlessAccount() || this.shouldShowCollectionThroughputInput());
```

Comment:
Does this mean it will also show collectionThroughput even for serverless accounts?

Reply:
No, this is only for showing the container vector policies and vector indexing policies editor. It's not used for showing collectionThroughput.